File: C program/ifelse_evenodd.c
#include<stdio.h>
void main()
{
	int i;
	printf("Enter number:");
	scanf("%d",&i);
	if(i%2==0)
	{
		printf("%d is even",i);
	}
	else
	{
		printf("%d is odd",i);
	}
}

File: Csharp_programs/ifelse_evenodd.cs
//2. Write a C# Sharp program to check whether a given number is even or odd.

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ifelse_evenodd
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int num;
            Console.WriteLine("Enter Number: ");
            num=Convert.ToInt32(Console.ReadLine());
            if(num%2==0)
            {
                Console.WriteLine("number is even:");
            }
            else
            {
                Console.WriteLine("number is odd:");
            }
            Console.ReadKey();
        }
    }
}
